src/hooks.ts

```diff
@@ -11,47 +11,31 @@ interface UsePositionConfig {
 }
 
 export function usePosition(config: UsePositionConfig) {
-  const { axisContext } = usePlotContext();
+  const { axisContext, width, height } = usePlotContext();
```

Comment:
It should use plotWidth and plotHeight. Width and height are the size of the entire SVG

Reply:
i think there is no need to height and width of the svg

Reply:
after last feat (the rectangle is drawn using x1 x2 y1 y2)

Reply:
sorry i didn't understand the first time, yes i checked the code and i'm changing it